src/app/controllers/user_sessions_controller.rb

```diff
@@ -7,7 +7,7 @@ def new
   end
 
   def create
-    @participant_session = ParticipantSession.new(participant_session_params)
+    @participant_session = ParticipantSession.new(participant_session_params.to_h)
```

Comment:
Any special reason for the `to_h`?

Reply:
I had to up authlogic to at least 4.2 based on:
https://github.com/binarylogic/authlogic#90-compatibility

Then, when running tests I bumped into this:
https://github.com/binarylogic/authlogic/blob/master/lib/authlogic/session/base.rb#L332